### ISSUE TITLE
Code analyzer results not reported in VSCode problems panel

### ISSUE THREAD

**Issue by rjgotten** [OmniSharp/omnisharp-roslyn#2411]
Omnisharp has stopped publishing Roslyn code analyzer results to the VSCode problems panel, unless opening individual files which happen to have a particular problem. The solution in question is quite large, so precautions were already taken to raise the diagnostic limit to 3000 files using `"csharp.maxProjectFileCountForDiagnosticAnalysis": 3000` 

Omnisharp.log confirms ~2400 files queued for analysis total, meaning this has more than enough leeway remaining:
```
[info]: OmniSharp.Roslyn.CSharp.Services.Diagnostics.CSharpDiagnosticWorkerWithAnalyzers
        Solution initialized -> queue all documents for code analysis. Initial document count: 2442.
```

While navigating to different individual files the extensions host log shows a spray of failures to execute `GetCodeActions`:

```
[2022-06-21 13:25:22.059] [exthost] [warning] DiagnosticCollection with name 'csharp' does already exist.
[2022-06-21 13:25:22.077] [exthost] [error] [ms-dotnettools.csharp] provider FAILED
[2022-06-21 13:25:22.077] [exthost] [error] Problem invoking 'GetCodeActions' on OmniSharp server: Error: Request /v2/getcodeactions cancelled, id: undefined
[2022-06-21 13:25:23.641] [exthost] [error] [ms-dotnettools.csharp] provider FAILED
[2022-06-21 13:25:23.641] [exthost] [error] Problem invoking 'GetCodeActions' on OmniSharp server: Error: Request /v2/getcodeactions cancelled, id: undefined
[2022-06-21 13:31:17.431] [exthost] [error] [ms-dotnettools.csharp] provider FAILED
[2022-06-21 13:31:17.431] [exthost] [error] Problem invoking 'GetCodeActions' on OmniSharp server: Error: Request /v2/getcodeactions cancelled, id: 69
[2022-06-21 13:36:06.795] [exthost] [error] [ms-dotnettools.csharp] provider FAILED
[2022-06-21 13:36:06.796] [exthost] [error] Problem invoking 'GetCodeActions' on OmniSharp server: Error: Request /v2/getcodeactions cancelled, id: 151
```

The omnisharp log shows warnings related to FindUsages:

```
[warn]: OmniSharp.Roslyn.CSharp.Services.Navigation.FindUsagesService
        No symbol found. File: <...>
```

where line numbers and column numbers make absolutely no sense; more often than not pointing to blank lines or comments.
Other than that no warnings or errors.

Technical details:
VSCode: 1.68.1
C# extension (`ms-dotnettools.csharp`): 1.25 (omnisharp server: 1.39.0-net6.0)
dotnet SDK:  .NET Core SDK 6.0.301 17.2.0

**Comment by rjgotten** [comment]
Crap. Probably misreported. Think this should've gone into omnisharp-vscode.
Can someone move it?

**Comment by rjgotten** [comment]
Never mind, turns out this was already reported as https://github.com/OmniSharp/omnisharp-vscode/issues/5171 back in april.
